src/locales/en.json

```
@@ -146,6 +146,8 @@
     "view": "View",
     "blog": "Read our blog",
     "howto": "Follow our guides",
-    "github": "Found an issue? Submit"
+    "github": "Found an issue? Submit",
+    "textNewLine": "Enter new line in text",
+    "textFinish": "Finish editing text"
```

```suggestion
    "textFinish": "Finish editing (text/multi-line)"
```

not sure about this one, but `enter` does more than confirm just text. In the future it may also confirm other things, though.